crates/nu-cli/src/shell/palette.rs

```diff
@@ -69,6 +69,8 @@ pub struct ThemedPalette {
 }
 
 impl ThemedPalette {
+    // remove this once we start actually loading themes.
+    `#[allow(dead_code)]`
```

Comment:
@jonathandturner I plan to use this on my next PR. is it ok if we ignore this dead code for now? If so, I think this PR is ready to go.